### DISCUSSION TITLE
How to programmatically enter select mode?

### DISCUSSION THREAD

**Discussion by morfert** [L3MON4D3/LuaSnip#888] [Q&A] [answered]
This is the only plugin that I know of that seems to use select mode. When you use a insert node with a placeholder it moves the cursor to the placeholder and selects it in select mode. I was hoping to do something in my config while entering select mode from a lua function but cannot find much documentation on how to do so.
Cheers

**Comment by L3MON4D3** [comment, marked as answer]
Unfortunately there's no pure lua-way to achieve this, we resort to feedkeys, you can find the implementation [here](https://github.com/L3MON4D3/LuaSnip/blob/ec7fba1d119fb5090a901eb616145450ffb95e31/lua/luasnip/util/util.lua#L165)

**Reply by morfert** [reply]
Yeah, feedkeys seems to work for my use case.
Thanks
